File: tmp-mei/fgenjs5030319945734616766/base/Abort_0.js
import { base$$Sealed_0 } from "../base/Sealed_0.js";
import * as rt from "../rt/main.js";

export class base$$Abort_0 extends base$$Sealed_0 {
  static $self = new base$$Abort_0();

  $exclamation$imm() {
  return base$$Abort_0.$exclamation$imm$fun(this);
}

  static $exclamation$imm$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}

File: tmp-mei/fgen4255027576711118260/base/Abort_0.java
package base;
public interface Abort_0 extends base.Sealed_0{
Abort_0 $self = new Abort_0Impl();
Object $exclamation$imm();
static Object $exclamation$imm$fun(base.Abort_0 $this) {
  return (switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (Object) null;
}})
;
}
}